mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactPdfView.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './ShowArtifactImageView.css';
+import { getSrc } from './ShowArtifactPage';
+import { Document, Page, pdfjs } from 'react-pdf';
+import { Pagination, Spin } from 'antd';
+import { getArtifactBytesContent } from './ShowArtifactUtils';
+import './ShowArtifactPdfView.css';
+
+class ShowArtifactPdfView extends Component {
+  constructor(props) {
+    super(props);
+    this.onPageChange = this.onPageChange.bind(this);
+    this.onDocumentLoadSuccess = this.onDocumentLoadSuccess.bind(this);
```

Comment:
```suggestion
```

These two lines are not necessary because both `onPageChange` and `onDocumentLoadSuccess` are arrow functions.

Reply:
Yup! Great catch - left that in there by accident while debugging